docs/DeveloperGuide.md

```diff
@@ -997,6 +1089,25 @@ testers are expected to do more *exploratory* testing.
 
 ---
 
+### Suggesting a show
+
+1. Prerequisites: None.
+2. Launch the app.
+3. Command: `suggest`
+  1. Test case:
```

Comment:
In our manual test cases, we should follow the format used, where it is `Test case: ` followed by the test case. In this case, it would be `Test case: Displayed list of shows is empty.`

Reply:
Updated Sir!